docs/changelog-fragments.d/1093.feature.md

```diff
@@ -0,0 +1,7 @@
+Added the ability to produce a json schema file for the settings file.
+
+```bash
+ansible-navigator settings --json-schema
```

Comment:
How about `ansible-navigator settings --schema-type jsonschema`? Provides scope to add newer schema type if required.

Reply:
done